### ISSUE TITLE
Hacktoberfest 2024 | Vertex AI 🤝 Workflows

### ISSUE THREAD

**Issue by PawelPeczek-Roboflow** [roboflow/inference#696]
# Vertex AI in Workflows
Are you ready to make a difference this Hacktoberfest? We invite you to contribute by integrating Google Vertex AI into our Workflows ecosystem! This powerful addition will enable users to harness advanced image and video analysis capabilities, enriching our platform and expanding its potential.

Join us in creating a seamless integration that empowers users to unlock insights from their visual data effortlessly. Whether you're a seasoned developer or new to open source, your contributions will play a crucial role in advancing our ecosystem. Let’s work together to bring this innovative functionality to life!

## Task description
* this task is open-ended - we do not have a fixed specification, apart from adoption of Vertex AI into Workflows ecosystem - we hope that open-source community will help us to figure out the best ways of adoptions
* we predict that this task may originate multiple different Workflow blocks - please share your plans and intentions to let other contributors know what you work on
* we are not sure how to solve authorization for Google Vertex AI - the inference server usually operates outside of gcloud, we would need to have secure way on injecting credentials in line with what Google proposes

## Cheatsheet 
* [Contributor guide](https://github.com/roboflow/inference/blob/main/CONTRIBUTING.md)
* [Workflows docs](https://inference.roboflow.com/workflows/about/)
* [Creating Workflow block - tutorial](https://inference.roboflow.com/workflows/create_workflow_block/)

**Comment by vanshpatelx** [comment]
@PawelPeczek-Roboflow is still open to contribute?

**Comment by PawelPeczek-Roboflow** [comment]
yeah - sure - if you are interested I am more than happy to help :)

**Comment by vanshpatelx** [comment]
Today I go through, all docs and code to understand how to implement. and anyway to connect you and your teams using discord or something like ?

**Comment by PawelPeczek-Roboflow** [comment]
let's use this communication channel for now
